dbms/src/Functions/IFunction.h

```diff
@@ -337,7 +339,7 @@ class FunctionBuilderImpl : public IFunctionBuilder
 
     DataTypePtr getReturnTypeWithoutLowCardinality(const ColumnsWithTypeAndName & arguments) const;
 };
-
+//#pragma GCC diagnostic pop
```

Comment:
Better to remove.

Reply:
The PR is far from being complete, I do go over my changes in the "final" commit, to remove all these leftovers